src/formElementPlugins/ApplicationViewWrapper.tsx

```diff
@@ -26,14 +26,14 @@ const ApplicationViewWrapper: React.FC<ApplicationViewWrapperProps> = (props) =>
     validate: (
       validationExpress: IQueryNode,
       validationMessage: string,
-      evaluator: Function
+      evaluatorParameters: EvaluatorParameters
     ): any => console.log('notLoaded'),
   })
   const [validationState, setValidationState] = useState<ValidationState>({} as ValidationState)
 
   useEffect(() => {
     if (!pluginCode) return
-    // TODO use generic or plugin specific
+    // TODO use plugin-specific validation method if defined
     setPluginMethods({
       validate: defaultValidate,
     })
```

Comment:
I don't understand exactly the idea with using the `useEffect` without any variable being watched. Can you add some comment on top of this one, so it's clear what this block is doing please?

Reply:
From [React Docs](https://reactjs.org/docs/hooks-effect.html)

> If you want to run an effect and clean it up only once (on mount and unmount), you can pass an empty array ([]) as a second argument. This tells React that your effect doesn’t depend on any values from props or state, so it never needs to re-run. This isn’t handled as a special case — it follows directly from how the dependencies array always works.
> 
> If you pass an empty array ([]), the props and state inside the effect will always have their initial values. While passing [] as the second argument is closer to the familiar componentDidMount and componentWillUnmount mental model, there are usually better solutions to avoid re-running effects too often. Also, don’t forget that React defers running useEffect until after the browser has painted, so doing extra work is less of a problem.

In other words, it means the Effect only runs once, but after all useState variables have been initialised -- which is important if you're going to `set` them within the useEffect.

Reply:
It's the equivalent of `componentDidMount` in class-based Compoments.

Reply:
Cool! Can you add a single comment on top stating `// Similar to componentDidMount`, please? I think that helps reading the code, at least to me.